server/tests/integrations/services/BaseServices/find.integration.test.ts

```diff
@@ -0,0 +1,33 @@
+import { closeDatabase, dataSource, startDatabase } from '../../../../src/db';
+import MemberServices from '../../../../src/services/MemberServices';
```

Comment:
relative path seems really not pratical, we should setup an alias to use absolute path froms src

Reply:
Implementing absolute path on PR #23